registry.md

```diff
@@ -7,35 +27,88 @@ This registry may be expanded with new RFCs.
 
 ## Ledgers
 
-<!-- TODO: Parameters to be moved in Bitcoin/Ethereum RFC -->
```

Comment:
I kind of liked the single table better. Too many sub-headers make it hard to read in markdown since the only difference is the font-size.

Reply:
The table is easier to read but it is deceptive. I think it only works because the way we specify both ledgers is so similar. What are you meant to do for multiple parameters? What about parameters where the type is not an enum type value. I found it very difficult to specify what I was adding in a particular RFC in a way that would be consistent.

Reply:
Okay, thank you for clarifying :)